from src.exceptions import new_exception

class AlphaVantageError(new_exception.NewException):
    pass